src/styles/blocks/card.css

```diff
@@ -0,0 +1,89 @@
+.card {
+  width: 200px;
+  height: 408px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  background: var(--background-color);
+  padding: 32px 16px 16px;
+}
+
+.card__img {
+  display: block;
+  width: 160px;
+  height: 134px;
+  margin: 0 auto 40px;
+}
+
+.card__title {
+  color: var(--main-color);
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+  margin: 0 0 4px;
+}
+
+.card__code {
+  color: var(--secondary-color);
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+  margin-bottom: 16px;
+}
+
+.card__reviews {
```

Comment:
use sass nesting

Reply:
I coded this in css not in scss so i cant use nesting here i guess